examples/common/start

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+set -euo pipefail
+
+minimum_kind_version=0.7.0
+
+. utils.sh
+
+print_usage() {
+  echo "Start Conjur Kubernetes demo on Kubernetes-in-Docker (KinD)."
+  echo "This includes:"
+  echo "  - Create a KinD cluster on your local machine."
+  echo "  - Helm install Conjur OSS on the KinD cluster."
+  echo "  - Load application-specific Conjur security policies."
+  echo "  - Deploy instances of a "Pet Store" application using the"
+  echo "    following Conjur authentication broker/clients:"
+  echo "      - Secretless Broker"
+  echo "      - Conjur Kubernetes Authen Client sidecar container"
+  echo "      - Conjur Kubernetes Authen Client init container"
+  echo
+  echo "Syntax:"
+  echo "$0 [arguments]"
+  echo
+  echo "Options:"
+  echo "-h                        Show help"
+  echo "-c <customize-env-file>   Customization file to use containing"
+  echo "                          environment settings. (Typically a"
+  echo "                          modified copy of 'customize.env'."
```

Comment:
Just a thought, feel free to take it or leave it...
We could potentially eliminate the platform conditional below by adding a `-e <platform-env-file>` flag (or `-p ...`) that would allow the platform-specific start script to tell this start script where its baseline environment settings are (i.e. `../openshift/0_export_env_vars.sh` or `../kubernetes-in-docker/0_export_env_vars.sh`).

I'm good either way.